.github/CONTRIBUTING.md

```diff
@@ -39,6 +39,11 @@ You'll see *exactly* what is sent, be able to add **your comments**, and then se
 * Create PRs that cover only **one specific issue/solution/bug**. Do not create PRs that are huge monoliths and could have been split into multiple independent contributions.
 * NewPipe uses [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor) to fetch data from services. If you need to change something there, you must test your changes in NewPipe. Telling NewPipe to use your extractor version can be accomplished by editing the `app/build.gradle` file: the comments under the "NewPipe libraries" section of `dependencies` will help you out.
 
+### Kotlin in NewPipe
+* NewPipeExtractor shall remain Java only
```

Comment:
I think this should go in the extractor's CONTRIBUTING.md, not here.

Reply:
the extractor has no contributing.md and is also mentioned here, so I thought it would be fitting here

Reply:
GitHub does not show a link to this when opening PRs on the extractor, so I think this still does not fit here and that a CONTRIBUTING.md should be added there.